menpofit/sdm/fitter.py

```diff
@@ -77,9 +77,8 @@ def _train(self, images, group=None, bounding_box_group=None,
 
         # In the case where group is None, we need to get the only key so that
         # we can attach landmarks below and not get a complaint about using None
-        first_image = first_batch[0]
```

Comment:
Hmm, it shouldn't be a copy... Although, in a batched scenario it may be that the landmarks are not properly added, due to the ``first_image`` referencing the first image of the first batch. But still, it shouldn't be a copy!